index.bs

```diff
@@ -7260,6 +7260,76 @@ The [=remote end steps=] given |session|, and |command parameters| are:
 
 </div>
 
+## Emulation ## {#module-emulation}
+
+The <dfn export for=modules>emulation</dfn> module contains functionality for
+emulating device-dependent and system-dependent parameters.
+
+### Definition ### {#module-emulation-definition}
+
+[=remote end definition=]
+
+<pre class="cddl remote-cddl">
+
+EmulationCommand = (
```

Comment:
If we're adding this as a new module, then you need to add it to the top-level `Command` definition.